packages/flutter/test/cupertino/switch_test.dart

```diff
@@ -526,6 +526,43 @@ void main() {
     expect(find.byType(CupertinoSwitch), paints..rrect(color: trackColor));
   });
 
+  testWidgets('Switch is using default thumb color', (WidgetTester
+  tester) async {
```

Comment:
This probably should be on the same line as `WidgetTester`.

Reply:
I will fix this

Reply:
done!